app/models/redde/url_generator.rb

```diff
@@ -11,9 +11,13 @@ def url
   end
 
   def translitted_name
-    Russian.translit(name).gsub(' ', '-')
-      .gsub(/[^\x00-\x7F]+/, '').gsub(/[^\w_ \-]+/i, '')
-      .gsub(/[ \-]+/i, '-').gsub(/^\-|\-$/i, '')
+    Russian
+      .translit(name)
+      .tr(' ', '-')
+      .gsub(/[^\x00-\x7F]+/, '')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.